stevedore-build-artifacts.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+artifact_path='producedbuilds/*'
+
+chmod -v +x Artifacts/Builds/Binaries/Linux/csc
+ls -l Artifacts/Builds/Binaries/Linux/csc
+chmod -v +x Artifacts/Builds/Binaries/Linux/VBCSCompiler
+ls -l Artifacts/Builds/Binaries/Linux/VBCSCompiler
+
+chmod -v +x Artifacts/Builds/Binaries/Mac/csc
+ls -l Artifacts/Builds/Binaries/Mac/csc
+chmod -v +x Artifacts/Builds/Binaries/Mac/VBCSCompiler
+ls -l Artifacts/Builds/Binaries/Mac/VBCSCompiler
+
+mkdir -p producedbuilds
+
+7z a producedbuilds/roslyn-csc-linux.7z $PWD/Artifacts/Builds/Binaries/Linux/*
+7z a producedbuilds/roslyn-csc-mac.7z $PWD/Artifacts/Builds/Binaries/Mac/*
+7z a producedbuilds/roslyn-csc-win64.7z $PWD/Artifacts/Builds/Binaries/Windows/*
+7z a producedbuilds/roslyn-csc-net46.7z $PWD/Artifacts/Builds/Binaries/Net46/*
```

Comment:
It occurs to me that this is not adding a LICENSE file to the artifacts (per the artifact guidelines, "All artifacts must contain a LICENSE file in their root"... even if our current csc artifacts are possibly also missing this). You can simply use the existing License.txt from the Roslyn repo (though the filename must be changed to uppercase): it should be enough to simply run
```cp License.txt $PWD/Artifacts/Builds/Binaries/Linux/LICENSE.txt```
(etc., for each of the four artifacts), _before_ 7z.